pandas/io/tests/test_excel.py

```diff
@@ -1328,6 +1328,20 @@ def test_to_excel_multiindex(self):
                             parse_dates=False)
             tm.assert_frame_equal(frame, df)
 
+    # GH13511
+    def test_to_excel_multiindex_nan_label(self):
+        _skip_if_no_xlrd()
+
+        frame = self.frame
+        frame.A = np.arange(len(frame))
+        frame.iloc[0, 0] = None
```

Comment:
I didn't add a test without NaNs, because there already are tests which cover this scenario, e.g. `test_to_excel_multiindex`, `test_to_excel_multiindex_cols` and `test_to_excel_multiindex_dates`.